R/thumbnail_color.R

```diff
@@ -37,6 +39,6 @@ thumbnail_color <- function(path, hex = TRUE) {
     res
   }) %>%
     set_names("red", "green", "blue", "alpha")
-  if (hex) result <- invoke(rgb, result)
+  if (hex) result <- rlang::exec(rgb, result)
```

Comment:
Great! Just remove the `::` 

```suggestion
  if (hex) result <- exec(rgb, result)
```